### ISSUE TITLE
Interactive elements not reachable via keyboard without TalkBack running [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#294]
The “Next”, “Previous”, and “Close” buttons on the “How it works” screens are not reachable when navigating the application via keyboard without TalkBack running. This means keyboard only users will not be able to access screens 2 -4 of "How it works".

![how-It-Works3 - Keyboard - Previous Next](https://user-images.githubusercontent.com/7034772/92118820-fb143580-edc4-11ea-8698-552d4f60b4f1.png)

![how-It-Works4 - Keyboard - Close](https://user-images.githubusercontent.com/7034772/92118836-ff405300-edc4-11ea-96bd-0d6d3197953d.png)

**Comment by jonnyjames** [comment]
@floridemai Just a note, the close ("X") button is now reachable using only the keyboard, but the "Previous", "Next" and "Close" buttons are not.

**Comment by ShaunBaker** [comment]
@jonnyjames Can you clarify what keyboard? Internal or external? And this is on a phone - not a tablet?

**Comment by jonnyjames** [comment]
@ShaunBaker Hi Shaun, this is using an external Bluetooth keyboard (specifically a Logitech K780) hooked up to an Android Pixel 2 phone.

**Comment by jonnyjames** [comment]
@floridemai @ShaunBaker This issue persists. What I'm seeing on the "How it works" screens is focus is still not moving to the previous/next (or close, on screen 4 of 4) buttons, but it does jump to the "Share" icon _behind_ the open overlay when using an external Bluetooth keyboard connected to the Android device.

**Comment by ckiss** [comment]
fixed, avail in next build

**Comment by jonnyjames** [comment]
This is issue is now fixed. Closing this issue.
